client/my-sites/plan-features/index.jsx

```diff
@@ -416,6 +417,7 @@ class PlanFeatures extends Component {
 						freePlan={ isFreePlan( planName ) }
 						isPlaceholder={ isPlaceholder }
 						isInSignup={ isInSignup }
+						manageHref={ `/plans/my-plan/${ site.slug }` }
```

Comment:
This isn't a blocker, but I'd prefer if we'd pass through the site slug vs the full object. It makes things a bit easier to see what the data needs of the component are.